lib/turn_junebug_expressway_web/utils.ex

```diff
@@ -49,4 +69,47 @@ defmodule TurnJunebugExpresswayWeb.Utils do
   def send_message(message) do
     TurnJunebugExpressway.MessageEngine.publish_message(message)
   end
+
+  def get_event_timestamp(event) do
+    event
+    |> Map.get("timestamp")
+    |> Timex.parse!("%Y-%m-%d %H:%M:%S.%f", :strftime)
+    |> DateTime.from_naive!("Etc/UTC")
+    |> DateTime.to_unix()
+    |> to_string()
+  end
+
+  def forward_event(payload) do
+    {:ok, event} = Jason.decode(payload)
+
+    status =
+      %{
+        "ack" => "submitted",
+        "nack" => "failed",
+        "delivery_report" => "delivered"
```

Comment:
A delivery report doesn't mean that it was delivered successfully. There are 3 kinds of delivery reports, delivery pending, succeeded, and failed.